src/cli.ts

```diff
@@ -45,6 +56,7 @@ export async function run() {
   let options = {
     tagFrom: argv["from"] || argv["tag-from"],
     tagTo: argv["to"] || argv["tag-to"],
+    nextVersion: argv["next-version"] === "" ? true : argv["next-version"],
```

Comment:
this seems to change the default behavior to "autodiscover", which would be a breaking change. the default (at least for now) should keep using "Unreleased".

Reply:
- when the parameter is not set, the value is undefined (**default**, the text will be "Unreleased")
- when the parameter is set without value, it equals *empty string*, which is typecasted as `true` (the lib will find the version by itself)
- when the parameter is set WITH a value, it uses it as the next version label

That's why I'm checking `argv["next-version"] === ""` instead of `!argv["next-version"]`